src/service/standard-actions-impl.js

```diff
@@ -71,7 +71,20 @@ export class StandardActions {
     switch (invocation.method) {
       case 'setState':
         bindForDoc(this.ampdoc).then(bind => {
-          bind.setState(invocation.args);
+          const args = invocation.args;
+          const objectString = args[OBJECT_STRING_ARGS_KEY];
```

Comment:
Discussion thread is here: https://github.com/ampproject/amphtml/pull/7573#discussion_r101643194